Applications/InSituLib/Adaptor.cpp

```diff
@@ -0,0 +1,82 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2017, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "Adaptor.h"
+
+#include <vtkCPDataDescription.h>
+#include <vtkCPInputDataDescription.h>
+#include <vtkCPProcessor.h>
+#include <vtkCPPythonScriptPipeline.h>
+#include <vtkNew.h>
+#include <vtkUnstructuredGrid.h>
+
+#include "MeshLib/Mesh.h"
+#include "MeshLib/Vtk/VtkMappedMeshSource.h"
+
+namespace InSituLib
+{
+vtkCPProcessor* Processor = NULL;
```

Comment:
@endJunction I tried with `vtkSmartPointer`, see the [diff](https://gist.github.com/bilke/6b08d31d8dd8f37f87b7763b9c191dd8) but got an exception at the end of the program which I was not able to fix.